core/src/apps/cardano/sign_tx.py

```diff
@@ -812,25 +800,22 @@ async def _show_output(
     if address_parameters := output.address_parameters:
         is_change_output = True
 
-        address_policy = get_address_policy(keychain, address_parameters)
-        assert address_policy == ADDRESS_POLICY_SHOW_SPLIT  # _should_show_output
-
-        payment_credential_policy = get_change_output_payment_credential_policy(
-            address_parameters
-        )
-        stake_credential_policy = get_change_output_stake_credential_policy(
+        payment_credential_policy = get_payment_credential_policy(address_parameters)
+        stake_credential_policy = get_stake_credential_policy(
             keychain, address_parameters
         )
 
         await show_credential(
             ctx,
             CredentialParams(CredentialParams.TYPE_PAYMENT, address_parameters),
             payment_credential_policy,
+            True,
```

Comment:
I'd rather include the name of the parameter in this case. The same applies also below. 
```suggestion
            is_change_output=True,
```